************* Module dbm.dumb
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:290:0: W0622: Redefining built-in 'open' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:33:0: C0103: Class name "error" doesn't conform to PascalCase naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:82:12: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:86:69: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:96:12: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:129:70: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:148:46: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:158:47: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:172:47: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:182:65: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:246:44: C0201: Consider iterating the dictionary directly instead of calling .keys() (consider-iterating-dictionary)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:254:12: R1720: Unnecessary "else" after "raise" (no-else-raise)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:198:8: W0201: Attribute '_modified' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:229:8: W0201: Attribute '_modified' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/dbm/dumb.py:307:8: C0103: Variable name "um" doesn't conform to snake_case naming style (invalid-name)

-----------------------------------
Your code has been rated at 9.15/10

